src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,24 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int MAX_LENGTH = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set charsFromFile = new TreeSet();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            int charValue;
+            while ((charValue = bufferedReader.read()) != -1) {
+                if (Character.isLetter(charValue)) {
+                    charsFromFile.add(Character.toLowerCase((char) charValue));
+                }
+            }
+        } catch (FileNotFoundException exception) {
+            throw new FileNotFoundException(exception.getMessage());
+        }
```

Comment:
you are catching and throwing the same type of exception. Let's remove catch block in this case